### ISSUE TITLE
Invalid version of kwnlp_sql_parser

### ISSUE THREAD

**Issue by jonwiggins** [kensho-technologies/kwnlp-preprocessor#7]
Trying to install the `main` branch seems to fail with
```
ERROR: Could not find a version that satisfies the requirement kwnlp_sql_parser<0.1,>=0.0.2 (from kwnlp-preprocessor) (from versions: 0.0.1)
ERROR: No matching distribution found for kwnlp_sql_parser<0.1,>=0.0.2
```

It looks like the most recent version of `kwnlp_sql_parser` is `0.0.1` (https://pypi.org/project/kwnlp-sql-parser/)

**Comment by DomenicPuzio** [comment]
Hey Jon, I'm sorry for the issue. I'll get the latest version published on PyPi as soon as I can. In the meantime, you can always download [the repo](https://github.com/kensho-technologies/kwnlp-sql-parser) and install from source.

**Comment by jonwiggins** [comment]
@DomenicPuzio Thanks for the help, I appreciate it.

It seems like there is currently an issue where the `kwnlp_sql_parser` doesn't return any rows when dumping the page table,
```python
>>> from kwnlp_sql_parser.wp_sql_dump import WikipediaSqlDump
>>> page_sql_dump = WikipediaSqlDump("/wikipedia-raw-20230201/pagetable/enwiki-20230201-page.sql.gz", allowlists={"page_namespace": ("0",)}, keep_column_names=("page_id","page_namespace","page_title",))
>>> page_sql_dump.to_csv("/out/test_page_dump.csv")
...
root:/out# cat test_page_dump.csv 
page_id,page_namespace,page_title
root:/out#
```

I tried to step through it a bit, and it looks like the call to `iter_matched_rows` (1) doesn't match any rows. Is this perhaps something that's fixed in the newer version?
I'm happy to open a PR on the other repo if you know what the issue is.

(1): https://github.com/kensho-technologies/kwnlp-sql-parser/blob/33f99eca684af5bd44d493717275d9dad09b365c/kwnlp_sql_parser/wp_sql_dump.py#L225

**Comment by DomenicPuzio** [comment]
Jon, this is exactly the issue fixed in version `0.0.2`. Give that a try and let me know if it works for you.

**Comment by DomenicPuzio** [comment]
Jon, I just wanted to close the loop here and let you know that `kwnlp-sql-parser` is available now! Let me know if you have any other issues.

https://pypi.org/project/kwnlp-sql-parser/0.0.2/
